getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -425,7 +425,8 @@ We want ``Player`` to detect when it's hit by an enemy, but we haven't made any
 enemies yet! That's OK, because we're going to use Godot's *signal*
 functionality to make it work.
 
-Add the following at the top of the script, after ``extends Area2D``:
+Add the following at the top of the script. If you're using GDScript, add it after
+``extends Area2D``. If you're using C#, add it after ``using System;``:
```

Comment:
For C# it needs to be inside the class declaration, the order does not really matter but by convention delegates are usually put at the top of the class. So it would be more like after ``public class Player : Area2D {``